core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -42,6 +43,8 @@
 public final class UnderFileSystemConfiguration extends InstancedConfiguration {
   private boolean mReadOnly;
   private boolean mShared;
+  private boolean mDetail;
+  private Recorder mRecorder;
 
```

Comment:
Do these have to go into the configuration? The verbosity option only controls how the command output is presented, and once the UFS is mounted, has no effect when interacting with the UFS, e.g. we are not using the recorder to record the calls to UFS.

Reply:
The purpose here is to pass the `recoder` through UnderFileSystemConfiguration. 
```java
void func1(UnderFileSystemConfiguration conf) {
  //...
    Recorder recorder = conf.getRecorder;
    recorder.recordIfEnabled("message");
    //...
}
```